README.md

```diff
@@ -1,18 +1,55 @@
 # Autocasting
-This plugin aims to notify the client when the player's magic level falls below the level 
-required for the autocast spell.
 
-This provides the player a chance to restore their magic level before attempting to 
-autocast again, which would normally mean that the autocasted spell would become 
-deselected and must be reselected in the autocast menu. 
+## Overview
+Autocasting is a plugin designed to help better manage your autocast spell. At its core, this plugin adds
+an overlay which dynamically updates information about your current autocast spell, including the amount of times
+you have runes to cast it. It will also alert when your autocast becomes unusable, due to your Magic dropping
```

Comment:
i might just be reading this wrong, but maybe `including the amount of casts you have left based on the runes in your inventory` is better?